### ISSUE TITLE
win:frame() fails in Hammerspoon 0.9.61

### ISSUE THREAD

**Issue by nloveladyallen** [Hammerspoon/hammerspoon#1728]
```lua
> win = hs.window.focusedWindow()
> f = win:frame()
...oon.app/Contents/Resources/extensions/hs/window/init.lua:298: ERROR: incorrect number of arguments. Expected 0, got 1
stack traceback:
	[C]: in function 'hs.window.internal._topLeft'
	...oon.app/Contents/Resources/extensions/hs/window/init.lua:298: in function 'hs.window.internal._frame'
	...oon.app/Contents/Resources/extensions/hs/window/init.lua:428: in function 'hs.window.internal.frame'
	[string "f = win:frame()"]:1: in main chunk
	[C]: in function 'xpcall'
	...app/Contents/Resources/extensions/hs/_coresetup/init.lua:417: in function <...app/Contents/Resources/extensions/hs/_coresetup/init.lua:397>
```

**Comment by latenitefilms** [comment]
This MIGHT be fixed in 0.9.62?

https://github.com/Hammerspoon/hammerspoon/releases/tag/0.9.62

**Comment by nloveladyallen** [comment]
Yes, it's fixed. Didn't see there was an update available.
